backup_mongo.sh

```diff
@@ -9,9 +9,47 @@ set -e
 # MONGO_AUTH_DB => name of mongo authentication database
 # AZURE_SA => Azure Storage account name
 # AZURE_BLOB_CONTAINER => name of the azure storage blob container
+# AZURE_SHARE_NAME => name of the azure file share
 # AZURE_DESTINATION_KEY => azure storage account destination key
 # DB => mongo db to backup
 
+# check the mongo uri
```

Comment:
I'd added checking for all the env vars since the container now runs when invoked